.github/workflows/release-drafter.yml

```diff
@@ -0,0 +1,14 @@
+name: Release Drafter
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  update_release_draft:
+    runs-on: ubuntu-20.04
```

Comment:
There is a deadline one ubuntu 20 for this year. Can we just use latest?

Reply:
Done :)